src/accounts/accountModel.java

```diff
@@ -0,0 +1,121 @@
+package accounts;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.ResourceBundle;
+import javafx.fxml.Initializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author aswecker
+ */
+public class accountModel implements Initializable {
+
+    private static final int ID = 1;
+    private static final int First_Name = 1;
+    private static final int Last_Name = 1;
+    private static final int Username = 1;
+    private static final int Password = 1;
+    
+    private static final String COMMA_DELIMITER = ",";
+    private static final String NEW_LINE = "\n";
+    private static final String FILE_HEADER = "Id,First_Name,Last_Name,Username,Password";
+    
+  
+
+   
+    @Override
+    public void initialize(URL url, ResourceBundle rb) {
+        addUser();
+        readUser();
+    }
+
+    // Add a user
+    public void addUser() {
+        User u = new User(1, "Test", "User", "TestUser", "password");
```

Comment:
Is this to set up our test users? Rather than do it in the accounts class, please create the accounts data file itself and put the data in there. Otherwise, every time this method is called these two users will be added to the file again.

Also, this method will need to take the user's information as parameters to write to the file (ie create the user).

Something else to consider is how to make sure that usernames and userIDs are unique. We can get away with assuming that the person creating the account would make sure the username is unique, but we do need to make sure the userID is unique. I ran into this with MoodRecord and used a method I learned from a person in a group for another class that might work here: using UUID class to generate a string.

`String recordId = UUID.randomUUID().toString();`